science/2022-11-23-science-blog-6/index.md

```diff
@@ -1,15 +1,261 @@
+
+In recent months, there has been criticism that in times of high incidence, a red warning is no longer meaningful. The risk of infection would exist regardless of whether there was a red warning in the CWA or not. Moreover, the frequency of red warnings is so high that the individual warning no longer has any significance. We want to show that this is not the case. In particular for rapid test results, which are often the first reason for a PCR test, the positivity rate with a red warning was significantly greater than with a green warning (low risk with risk exposure). Furthermore, we investigate whether these greater positivity rates with a red warning can be solely explained by local fluctuations in incidence.
 
 <!-- overview -->
 
-An English version of this science blog article will follow in due course. Please take a look at the [German version](/de/science/2022-11-23-science-blog-6) until the English version is available.
\ No newline at end of file
+1. [The most important findings in a nutshell: numbers, data and facts](#1-the-most-important-findings-in-a-nutshell-numbers-data-and-facts)
+2. [Positivity rates (nationwide)](#2-positivity-rates-nationwide)
+    1. [RAT](#21-rat)
+    2. [PCR](#22-pcr)
+3. [Local differences in incidence](#3-local-differences-in-incidence)
+4. [Positivity rates (local))](#4-positivity-rates-local)
+    1. [RAT](#41-rat)
+    2. [PCR](#42-pcr)
+5. [Positivity rate and incidence](#5-positivity-rate-and-incidence)
+    1. [RAT](#51-rat)
+    2. [PCR](#52-pcr)
+6. [Differences between urban and rural districts](#6-differences-between-urban-and-rural-districts)
+7. [Conclusion](#7-conclusion)
+8. [Continued further](#8-continued-further)
+
+## 1 The most important findings in a nutshell: numbers, data and facts
+
+- Whether rapid antigen test or PCR test, the positivity rate of red warnings is always greater than that of green warnings (low risk with risk exposure). In other words, where risk exposures are detected, the CWA can reliably assess the risk.
+
+- For rapid antigen tests, the positivity rate (the proportion of all tests performed that are positive) after a red warning is at all times greater than the positivity rates after a green warning (lower risk with risk exposure) or low risk without risk exposure.
+
+- For PCR tests, the positivity rate after a red warning is always greater than the positivity rates after a green warning (lower risk with risk exposure). In times of high incidence, the positivity rate after a red warning is in part smaller than at lower risk without risk exposure - here a strong preselection by positive antigen rapid tests is to be assumed.
```

Comment:
I don't understand the second sentence "than at lower risk without risk exposure". I also didn't understand it properly in the German version.